specials/SPC18_Matias_SpiesVsMercs.Altis/initPlayerLocal.sqf

```diff
@@ -1,2 +1,14 @@
 #include "do_not_edit\script_component.hpp"
 #include "do_not_edit\initPlayerLocal.sqf"
+
+["mission_alarm", {
+  playSound "Alarm";
+}] call CBA_fnc_addEventHandler;
+
+["mission_hackAbort", {
+  playSound "ACE_Javelin_Locked";
```

Comment:
Do you want a global sound for this? A player may cancel hacking in order to remain concealed, but this will give them away